README.md

```diff
@@ -123,9 +123,14 @@ To test changes from packages in this repository in other applications follow th
   rm -rf node_modules/react; rm -rf node_modules/react-dom
   ```
   This is because we want to use hooks and different reacts are not playing nicely with hooks [facebook/react/issues/15315](https://github.com/facebook/react/issues/15315)
+3. Link `react` and `react-dom` from your application.
+```
+ln -s $PWD/insights-inventory-frontend/node_modules/react frontend-components/node_modules/react
```

Comment:
Maybe I would use some placeholder` $PWD/<application-folder>/...`

Reply:
+1 and possible use something like `/path/to/frontend-components/...` for the target to not confuse people thinking it is implied that it is within the apps directory. I don't think anyone would think that, but to be sure. :)